include/mbedtls/aria.h

```diff
@@ -61,11 +61,11 @@ extern "C" {
 /**
  * \brief The ARIA context-type definition.
  */
-typedef struct mbedtls_aria_context
-{
+typedef struct mbedtls_aria_context {
     unsigned char MBEDTLS_PRIVATE(nr);           /*!< The number of rounds (12, 14 or 16) */
     /*! The ARIA round keys. */
-    uint32_t MBEDTLS_PRIVATE(rk)[MBEDTLS_ARIA_MAX_ROUNDS + 1][MBEDTLS_ARIA_BLOCKSIZE / 4];
+    uint32_t MBEDTLS_PRIVATE(rk)[MBEDTLS_ARIA_MAX_ROUNDS +
+                                 1][MBEDTLS_ARIA_BLOCKSIZE / 4];
```

Comment:
Can we do something about this? :D

Reply:
We'd much prefer
```
foo[a @ b]
   [c @ d];
```
Multidimensional arrays are pretty rare in our code base, so if uncrustify doesn't have an obvious rule to favor this, we can hopefully guide it by adding the line break ourselves.